<launch>

    <node name="tracking_sast2" pkg="sensors" type="tracking_sast2" >
	<param name = "kmin_x" value = "0.001" />
	<param name = "kmin_y" value = "0.001" />
	<param name = "mu_x" value = "0.1" />
	<param name = "mu_y" value = "0.1" />
    <param name = "lambda_x" value = "0.001" />
	<param name = "lambda_y" value = "0.001" />

	<param name = "kmin_u" value = "0.1" />
	<param name = "kmin_r" value = "0.1" />
	<param name = "mu_u" value = "0.01" />
	<param name = "mu_r" value = "0.01" />
    <param name = "lambda_u" value = "0.001" />
	<param name = "lambda_r" value = "0.001" />
    </node>

</launch>
